text/0001-remove_ambiguous_grammar.md

```diff
@@ -0,0 +1,234 @@
+# SurrealQL: Remove ambiguity in grammar
+
+Status: Open for suggestion
+Creator: Mees Delzenne
+Owner: Mees Delzenne
+Last edited time: October 11, 2023 3:56 PM
+tag: SurrealQL
+
+# 1. Summary
+
+The current version of SurrealQL grammar, as defined by what the parser currently accepts, contains several ambiguous productions. These productions are parsed differently depending on the context or can seem very similar to each other, but subtle differences can result in completely different semantics.
```

Comment:
question: what is a "production".

Reply:
Never mind, I see the glossary.  Maybe bring Glossary to top, or have a table of contents?

Reply:
Alright! will do!